api/src/main/resources/workbench-api.yaml

```diff
@@ -6770,12 +6770,15 @@ definitions:
 
   InstitutionMembershipRequirement:
     type: string
-    description: 'Type of Institution requirement for member to access a tier.
-      It can be matching email address, email domains or no access'
+    description: 'Institution email matching requirements for tier access.
+      We can match on email DOMAINS or individual email ADDRESSES.  NO_ACCESS indicates that the
+      institution does not provide access to the tier at all, and UNINITIALIZED is for unsaved
+      institutions in the admin UI.'
     enum:
       - DOMAINS
       - ADDRESSES
       - NO_ACCESS
+      - UNINITIALIZED
```

Comment:
this is a new state for UI only which replaces `null`.  If we attempt to create an Inst with this state, the API returns an error.